src/pages/studyView/charts/barChart/BarChart.tsx

```diff
@@ -140,15 +141,21 @@ export default class BarChart extends React.Component<IBarChartProps, {}> implem
     @computed
     get bottomPadding(): number {
```

Comment:
Let's also move the `10px` reserved length for the height to here as well. Let the height and the padding in sync.

Reply:
@zhx828 Thank you for the review. I am working on these comments now. I am using chrome browser on ubuntu 18.04. I take the screenshot with `$gnome-screenshot`. The bars look blue in my browser, but purple on the screenshot.